packages/frontend/src/components/Search/FilterSearch.tsx

```diff
@@ -23,69 +23,149 @@ import { DatePicker } from '@mui/x-date-pickers';
 import { LocalizationProvider } from '@mui/x-date-pickers/LocalizationProvider';
 import { AdapterDateFns } from '@mui/x-date-pickers/AdapterDateFns';
 
-import { SearchParameterName, FilterSearchData, IItem } from './FilterSearchData';
+import { Sort, useFiltersStore } from './filterStore';
+import { Area, areas, categories } from '../../utils/categories';
+import { Category } from './FilterSearchData';
 import { ExpandLess, ExpandMore } from '@mui/icons-material';
 import { Colors } from '../../constants/Colors';
-import { SearchContext } from '../../contexts/SearchContext';
 import { ButtonWrapper } from '../../styles/common';
 import { useTranslation } from 'react-i18next';
-import { EMimeType, SortDataType } from '../../constants/Data';
+import { FileFormats, SortDataType } from '../../constants/Data';
 import { AppBarContext } from '../../contexts/AppBarContext';
-import { mapSortTypeToValue } from '../../utils/helpers';
 
-interface IFilterSearchItem extends IItem {
-  checkboxes: any;
-  handleCheckboxes: any;
+interface CategoryFilterProps {
+  title: string;
+  categories?: Category[];
 }
 
-const FilterSearchItem = (props: IFilterSearchItem) => {
+const AreaFilter = () => {
   const { t } = useTranslation(['search']);
   const [open, setOpen] = useState(false);
-  const { savedCheckboxes } = useContext(SearchContext);
+  const activeArea = useFiltersStore((state) => state.area);
+  const updateArea = useFiltersStore((state) => state.updateArea);
 
-  const handleClick = () => {
-    setOpen((prevState: boolean) => !prevState);
+  const handleAreaUpdate = (area: Area) => {
+    if (area.area === activeArea?.area) {
+      updateArea(null);
+      return;
+    }
+    updateArea(area);
   };
 
-  const isDefaultChecked = (name: SearchParameterName, value: string) => {
-    return savedCheckboxes[name]?.indexOf(value) !== -1;
-  };
+  return (
+    <>
+      <ListItem key={'categories'} onClick={() => setOpen(!open)}>
+        <ListItemText
+          disableTypography
+          primary={
+            <Typography variant="body1" textTransform="uppercase" sx={{ color: Colors.darkgrey }}>
+              {t('search:area')}
+            </Typography>
+          }
+        />
+        {open ? <ExpandLess /> : <ExpandMore />}
+      </ListItem>
+      <Collapse in={open} timeout="auto" unmountOnExit>
+        <List component="li" disablePadding>
+          {areas?.map((area: Area) => {
+            return (
+              <ListItem>
+                <Radio
+                  defaultChecked={false}
+                  checked={area.area === activeArea?.area}
+                  onClick={() => handleAreaUpdate(area)}
+                />
+                <ListItemText primary={area.title} style={{ textTransform: 'capitalize' }} />
```

Comment:
Areas seem to be are missing nordic letters here. Add them to the constants where these areas and their ids are defined or use `getReadableAreaTitle(area.name)` here?